snap7/common.py

```diff
@@ -21,7 +23,7 @@ class ADict(dict):
     Accessing dict keys like an attribute.
     """
     __getattr__ = dict.__getitem__
-    __setattr__ = dict.__setitem__
+    __setattr__ = dict.__setitem__  # type: ignore
```

Comment:
why does this require ignoring?

Reply:
Same here, I will reproduce the error later and show it

Reply:
mypy? but that is strange, we were testing with mypy before also.

Reply:
Here is the error.
````
snap7/common.py:26: error: Incompatible types in assignment (expression has type "Callable[[Dict[_KT, _VT], _KT, _VT], None]", base class "object" defined the type as "Callable[[object, str, Any], None]")
```